### PR TITLE
Changes in the course and program

### PR DESCRIPTION
**Pull request by manassolanki**
https://github.com/frappe/erpnext/issues/8367

prcourse

### CODE REVIEW

File: erpnext/schools/doctype/course/course.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
 frappe.ui.form.on("Course", "refresh", function(frm) {
 	if(!cur_frm.doc.__islocal) {
 		frm.add_custom_button(__("Program"), function() {
-			frappe.route_options = {
-				"Program Course.course": frm.doc.name
-			}
+			// frappe.route_options = {
```
Comment (nabinhait) [inline erpnext/schools/doctype/course/course.js]: remove commented code

File: erpnext/schools/doctype/course/course.json [hunk 2]
```diff
@@ -65,6 +66,35 @@
...
+   "set_only_once": 0, 
+   "unique": 0
+  }, 
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "parent_course", 
+   "fieldtype": "Data",
```
Comment (nabinhait) [inline erpnext/schools/doctype/course/course.json]: Why not Link?

File: erpnext/schools/doctype/course/course.js (hunk 1 above)
Comment (manassolanki) [inline erpnext/schools/doctype/course/course.js]: Because I have to add this after merging the student batch and student group.

**Comment by nabinhait** [comment]
@manassolanki 
Describe what changes have you made, check https://github.com/frappe/erpnext/wiki/Pull-Request-Guidelines
And check inline comments.

**Pull request closed without merging**